src/Appwrite/Platform/Modules/Databases/Http/Databases/Collections/Documents/Create.php

```diff
@@ -350,6 +350,20 @@ public function action(string $databaseId, string $documentId, string $collectio
 
             // Assign a unique ID if needed, otherwise use the provided ID.
             $document['$id'] = $sourceId === 'unique()' ? ID::unique() : $sourceId;
+
+            // Allowing to add createdAt and updatedAt timestamps if server side(api key)
+            $createdAt = $document['$createdAt'] ?? null;
+            $updatedAt = $document['$updatedAt'] ?? null;
```

Comment:
If we're not using these variables later, let's just do the check inline instead